### DISCUSSION TITLE
Compile as shared library like DLL

### DISCUSSION THREAD

**Discussion by yesilcimenahmet** [cesanta/mongoose#2086] [Ask questions, get answers] [answered]
Hi, 
I want to compile Mongoose as library to port for other library such as Delphi, Can I compile Mongoose lib as DLL? but I could not do that for real. Is it any info about that ?

Thanks.

**Comment by yesilcimenahmet** [comment]
any road map pls?

**Comment by scaprile** [comment, marked as answer]
#2091
